refinery/tool_manager/tests.py

```diff
@@ -111,40 +114,44 @@ def test_for_proper_parameters_response(self):
 
 class ToolDefinitionGenerationTests(TestCase):
     def test_workflow_improperly_annotated(self):
-        with open("tool_manager/test-data/workflow_annotation_invalid.json",
-                  "r") as f:
+        with open("tool_manager/test-data/workflow_annotation_invalid"
```

Comment:
It's probably a good idea for consistency. All other folder and file names appear to have underscores.